src/routes/news/containers/TunewsPage.js

```diff
@@ -48,6 +54,17 @@ export class TunewsPage extends React.PureComponent<PropsType> {
 
   render () {
     const { tunews, language, city, t, fetchMoreTunews, hasMore, isFetchingFirstTime, isFetching, cities } = this.props
+
+    if (this.props.areCitiesFetching) {
+      return <LoadingSpinner />
+    }
+
+    const currentCity: CityModel = cities && cities.find(cityElement => cityElement.code === city)
+
+    if (!currentCity.tunewsEnabled) {
```

Comment:
As discussed: A `ContentNotFoundError` should be displayed here.

Reply:
Fixed!